apps/nextjs-website/react-components/components/HowTo/HowTo.tsx

```diff
@@ -86,26 +90,34 @@ const HowTo = (props: HowToProps) => {
 
         {/** Link */}
         {link && (
-          <Link
-            sx={{
-              display: 'flex',
-              alignItems: 'center',
-              mt: 6,
-              alignSelf: 'flex-start',
-            }}
-            href={link.href}
-            target={link.target}
-            color={textColor}
-            underline='none'
-            fontWeight={600}
+          <Typography
+            component='span'
+            variant='body1'
+            color={theme === 'light' ? 'text.primary' : 'background.paper'}
+            fontSize={'18px'}
+            display={'contents'}
```

Comment:
Is this necessary?

Not implying it isn't, just asking the reason why it's been used.

Reply:
only display={'contents'} was needed for link alignment, the rest was removed